src/executables/sysrepo-plugind.c

```diff
@@ -502,10 +488,7 @@ write_pidfile(int pidfd)
         return -1;
     }
 
-    if (snprintf(pid, sizeof(pid) - 1, "%ld\n", (long) getpid())) {
-        error_print(0, "Failed to allocate memory for pid (%s).", strerror(errno));
-        return -1;
-    }
```

Comment:
A patch about "notification aging" should not remove an error handler for a`snprtinf` in an unrelated function.